[2025-02-11T18:25:05.852+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T18:25:05.868+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: superstore_dag.run_dbt_dim_date scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T18:25:05.877+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: superstore_dag.run_dbt_dim_date scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T18:25:05.878+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-11T18:25:05.891+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): run_dbt_dim_date> on 2025-02-10 00:00:00+00:00
[2025-02-11T18:25:05.899+0000] {standard_task_runner.py:72} INFO - Started process 623 to run task
[2025-02-11T18:25:05.901+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'superstore_dag', 'run_dbt_dim_date', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/superstore_dag.py', '--cfg-path', '/tmp/tmppq19tqgh']
[2025-02-11T18:25:05.910+0000] {standard_task_runner.py:105} INFO - Job 203: Subtask run_dbt_dim_date
[2025-02-11T18:25:05.936+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-11T18:25:05.989+0000] {task_command.py:467} INFO - Running <TaskInstance: superstore_dag.run_dbt_dim_date scheduled__2025-02-10T00:00:00+00:00 [running]> on host 9cd0ccfaeb2e
[2025-02-11T18:25:05.991+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-11T18:25:06.070+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='superstore_dag' AIRFLOW_CTX_TASK_ID='run_dbt_dim_date' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-11T18:25:06.070+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T18:25:06.084+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-11T18:25:06.085+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /dbt/superstore/ && dbt run --select dim_date']
[2025-02-11T18:25:06.094+0000] {subprocess.py:99} INFO - Output:
[2025-02-11T18:25:08.195+0000] {subprocess.py:106} INFO - [0m18:25:08  Running with dbt=1.9.2
[2025-02-11T18:25:08.478+0000] {subprocess.py:106} INFO - [0m18:25:08  Registered adapter: postgres=1.9.0
[2025-02-11T18:25:08.783+0000] {subprocess.py:106} INFO - [0m18:25:08  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-02-11T18:25:08.784+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-02-11T18:25:08.784+0000] {subprocess.py:106} INFO - - models.superstore_project.dim_product
[2025-02-11T18:25:08.975+0000] {subprocess.py:106} INFO - [0m18:25:08  Found 5 models, 8 data tests, 1 source, 431 macros
[2025-02-11T18:25:08.978+0000] {subprocess.py:106} INFO - [0m18:25:08
[2025-02-11T18:25:08.980+0000] {subprocess.py:106} INFO - [0m18:25:08  Concurrency: 1 threads (target='prod')
[2025-02-11T18:25:08.982+0000] {subprocess.py:106} INFO - [0m18:25:08
[2025-02-11T18:25:09.082+0000] {subprocess.py:106} INFO - [0m18:25:09  1 of 1 START sql incremental model core.dim_date ............................... [RUN]
[2025-02-11T18:25:09.295+0000] {subprocess.py:106} INFO - [0m18:25:09  1 of 1 OK created sql incremental model core.dim_date .......................... [[32mINSERT 0 964[0m in 0.21s]
[2025-02-11T18:25:09.317+0000] {subprocess.py:106} INFO - [0m18:25:09
[2025-02-11T18:25:09.318+0000] {subprocess.py:106} INFO - [0m18:25:09  Finished running 1 incremental model in 0 hours 0 minutes and 0.33 seconds (0.33s).
[2025-02-11T18:25:09.431+0000] {subprocess.py:106} INFO - [0m18:25:09
[2025-02-11T18:25:09.432+0000] {subprocess.py:106} INFO - [0m18:25:09  [32mCompleted successfully[0m
[2025-02-11T18:25:09.432+0000] {subprocess.py:106} INFO - [0m18:25:09
[2025-02-11T18:25:09.433+0000] {subprocess.py:106} INFO - [0m18:25:09  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-11T18:25:10.335+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-11T18:25:10.361+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T18:25:10.362+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=superstore_dag, task_id=run_dbt_dim_date, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250211T182505, end_date=20250211T182510
[2025-02-11T18:25:10.389+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T18:25:10.408+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T18:25:10.410+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
